modules/installation-gcp-user-infra-adding-ingress.adoc

```diff
@@ -27,15 +31,11 @@ endif::[]
 
 .Prerequisites
 
-* Configure a GCP account.
+* Ensure you installed a cluster.
```

Comment:
Suggest to remove "Ensure you installed a cluster." as a prerequisites, because we'll have "Ensure the bootstrap process completed successfully.", thanks!

Reply:
@brendan-daly-red-hat Other parts look good to me, thanks!